_posts/2020-06-16-aws-acronyms-overview.md

```diff
@@ -0,0 +1,154 @@
+The server for executing the function is provisioned at the time of invocation. The infrastructure is dynamically scaled, depending on the number of concurrent requests. 
+**Lambda is commonly invoked by events from other AWS services like API Gateway, SQS, SNS, or Cloudwatch**.
+
+**SAM-Serverless Application Model is the framework for developing lambda applications with useful tools like a CLI, a local test environment based on docker, and integration with developer tools.** 
+ 
+## Deliver Content With CloudFront
+[AWS CloudFront](https://aws.amazon.com/cloudfront) is a CDN-content delivery network service to serve content from the nearest AWS POP-point of presence. **It is a global autoscaled service integrated for content origin with resource endpoints in EC2, ELB, and S3.** 
+
+## Route To Your IP With Route 53 As DNS
+[AWS Route 53](https://aws.amazon.com/route53) is the DNS-Domain Name System service with capabilities of high availability and scalability. **It provides ways to route incoming end-user requests to resources within AWS like EC2, ELB load balancer and also to resources outside AWS using a group of routing rules based on network latency, geo-proximity, and weighted round-robin.**
+
+
+## Governance, Compliance & Audit With CloudTrail
+Security in the cloud works on the principle of shared responsibility something you will find repeated ad-nausea across AWS literature. AWS is responsible for the security of the cloud and we are responsible for security in the cloud. 
+
+[CloudTrail](https://aws.amazon.com/cloudtrail) is a service that is switched on by default in an AWS account but we need to build controls to ensure nobody switches it off, modifies the generated trails, sends trails to an S3 bucket accessible to our security teams.
+
+**CloudTrail helps to gain complete visibility into all user activity in the form of events telling you who did what and when. It provides event history of all the activities done in your AWS account.**
+
+## Observability With CloudWatch
+With the advent of distributed applications, observability has emerged as a key capability to monitor the health of systems and identify the root cause of problems like outage or slowness. [CloudWatch](https://aws.amazon.com/cloudwatch) is understandably among the first services from AWS stable.
+
+**AWS CloudWatch comprises services for logging, monitoring, and event handling**. We send logs from various AWS resources like EC2 and even our applications to CloudWatch. Resources also emit a set of metrics over which we create alarms, for example, CPU above 80% to enable us to take remedial actions. CloudWatch Events rechristened EventBridge allows us to configure remedial actions in response to any events of our interest.
+ 
+## Conclusion With Mind Map
+ I have put everything together in the mind map below.
+
+ ![AWS Mind Map](/assets/img/posts/aws-acronyms-overview/mindmapaws.png)
```

Comment:
I love the mind map! I'll see if I can even use it as the cover image for this article. 

Could you make it with white background instead of transparent background? This way we could create a link to the full version of the image here and it would still be readable (in my case, it would show on a black browser background, which makes it less readable).